stackable-base/Dockerfile

```diff
@@ -4,7 +4,28 @@
 # https://catalog.redhat.com/software/containers/ubi9/ubi-minimal/615bd9b4075b022acc111bf5?image=6633b5136d81a5634a616c41&architecture=amd64
 
 # Manifest list digest because of multi architecture builds ( https://www.redhat.com/architect/pull-container-image#:~:text=A%20manifest%20list%20exists%20to,system%20on%20a%20specific%20architecture )
-FROM registry.access.redhat.com/ubi9/ubi-minimal@sha256:a7d837b00520a32502ada85ae339e33510cdfdbc8d2ddf460cc838e12ec5fa5a AS builder
+FROM registry.access.redhat.com/ubi9/ubi-minimal@sha256:a7d837b00520a32502ada85ae339e33510cdfdbc8d2ddf460cc838e12ec5fa5a AS product-utils-builder
+
+ARG CONFIG_UTILS_VERSION=main
+
+RUN microdnf update --assumeyes && \
+    microdnf --assumeyes install \
+    # Needed to fetch source
+    git \
+    # Needed for compilation
+    gcc && \
+    microdnf clean all
+
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y --default-toolchain 1.78.0 && \
+    . "$HOME/.cargo/env" && cargo install cargo-cyclonedx@0.4.0 cargo-auditable@0.6.2
```

Comment:
I know we don't do this in the other builder but could you move all the version numbers to environment variables? That might allow us to actually use Renovate for this.

https://docs.renovatebot.com/modules/manager/regex/#regular-expression-capture-groups

Reply:
I got it working without environment variables but it is definitely nicer to have env variables. Makes for a more robust regex.

Reply:
@lfrancke I pushed some changes, but I have no idea how to test it. Happy about any comments :)

Reply:
Lol I thought renovate would pick my comments up, but it seems like we still need the regex thing...

Reply:
Yes, we do. I didn't even expect you to include the comments, only the env vars. I'll have to check those out in detail. Sorry.

If you want to merge this fast I'm happy to approve a version without the renovate comments. If you're happy to merge later I can check those out when I have time.

Reply:
Ok, as I'm no expert on renovate, I removed the comments again and would be happy to address it later on :sweat: 
But I would push a commit were I do the same fix for the rust images (just for cleanup)